### PR TITLE
Ext init phase

### LINKED ISSUE #1818: EXTENSIBILITY: Extensions have an init phase to detect if user has access to the resource underneath
**Description**

Extensions even if present can be disabled if the user has no access to them.

**Reasons**

**Attachments**

### PR DESCRIPTION
**Pull request by wchrapka**
**Description**
Hides resources that don't exists or user doesn't have persmission for 

**Related issue(s)**

Resolves #1818

### CODE REVIEW

File: core/src/assets/customResources/customResources.json [hunk 1]
```diff
@@ -1,81 +1 @@
...
-        "spec.exportTo": "Exported To Namespaces",
-        "spec.host": "Host",
-        "spec.trafficPolicy": "Traffic Policy",
-        "spec.subsets": "Subsets",
-        "spec.workloadSelector": "Workload Selector"
-      }
-    }
-  }
-}
+{}
```
Comment (dariadomagala) [inline core/src/assets/customResources/customResources.json:1]: I believe we should move this resource to a config map in examples

**Review by dariadomagala — DISMISSED** [review]

**Review by Wawrzyn321 — APPROVED** [review]

**Pull request merged**